.github/workflows/python-package.yml

```diff
@@ -1,74 +0,0 @@
-# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
```

Comment:
I am not sure why the whole CI is deleted?

Reply:
It's just the comment. Not the entire file.